docs/cosmology/io.rst

```diff
@@ -65,12 +62,154 @@ method. The method uses this cosmology class, regardless of the class
 indicated in the file, and sets parameters' default values from the class'
 signature.
 
-::
+.. doctest-skip::
 
     >>> from astropy.cosmology import FlatLambdaCDM
     >>> cosmo = FlatLambdaCDM.read('[file name]')
     >>> cosmo == Planck18
     True
+    
+Reading and writing :class:`~astropy.cosmology.Cosmology` objects go through
```

Comment:
Done! I include an example of ``.list_formats()`` and also a table of the available built-in formats, since I note that ``.list_formats`` can include registered 3rd party formats.